articles/videos/get-started/02-provision-user-stores.md

```diff
@@ -0,0 +1,190 @@
+---
+description: How user profiles are provisioned within an Auth0 tenant.
+classes: video-page
+---
+# Provision: User Stores
+
+Learn how user profiles are provisioned within an Auth0 tenant. Users can come from a multitude of places: directly through creating an account using self sign up, indirectly through using their favorite trusted social media platform, or collectively through a trusted enterprise directory. Auth0 makes it simple to quickly connect your tenant to multiple types of user stores. We group these users stores into three categories: database, social, and enterprise.
```

Comment:
We always seem to leave off legal identity as a user store. Maybe this doesn't matter, but feels like it's something missing, particularly for European users.

Reply:
I'll add this to the list for the next revision.